packages/cli/lib/sandboxes.js

```diff
@@ -81,6 +86,43 @@ async function getAvailableSyncTypes(parentAccountConfig, config) {
   return syncTypes.map(t => ({ type: t.name }));
 }
 
+/**
+ * @param {Object} accountConfig - Account config of sandbox portal
+ * @param {Array} availableSyncTasks - Array of available sync tasks
+ * @param {Boolean} skipPrompt - Option to skip contact records prompt and return all available sync tasks
+ * @returns {Array} Adjusted available sync task items
+ */
+const getSyncTypesWithContactRecordsPrompt = async (
+  accountConfig,
+  availableSyncTasks,
+  skipPrompt = false
+) => {
+  // Fetches sync types based on default account. Parent account required for fetch
+  let syncTasks = availableSyncTasks;
```

Comment:
Why the need for this?

Reply:
For line 101, we filter the types based on the prompt on line 120. If no changes, we just return that set of tasks. Do you mean we can just ditch the var and filter on the param passed in?

Reply:
> Do you mean we can just ditch the var and filter on the param passed in? 

Yes, that's what I was thinking, although this is really a nit. `syncTasks` has better readability, so I'm happy to keep it if you are!

Reply:
I've removed and pushed it into the latest branch #833 !